test/e2e_node/critical_pod_test.go

```diff
@@ -147,6 +147,66 @@ var _ = SIGDescribe("CriticalPod", framework.WithSerial(), framework.WithDisrupt
 	})
 })
 
+var _ = SIGDescribe("CriticalPodWithPodLevelResources", framework.WithSerial(), framework.WithDisruptive(), feature.PodLevelResources, func() {
```

Comment:
Since this is the first addition of Node E2E tests related to Pod Level Resources feature, https://github.com/kubernetes/test-infra/pull/35061 seems necessary to pass presubmit job (`pull-kubernetes-node-e2e-containerd-alpha-features`).

Reply:
Ah, sorry. I understand now. The tests under `test/e2e_node` are also run in `pull-kubernetes-node-kubelet-serial-podresources`. Never mind.
- https://github.com/kubernetes/test-infra/blob/bc5ddfd4bf280b996e9667457169557e60c7e44b/config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml#L890-L946
- https://testgrid.k8s.io/sig-node-presubmits#pr-kubelet-serial-e2e-podresources